gems/pending/spec/disk/modules/azure_blob_disk_spec.rb

```diff
@@ -0,0 +1,152 @@
+require 'ostruct'
+require 'azure-armrest'
+require 'disk/modules/AzureBlobDisk'
+require 'vcr'
+
+describe AzureBlobDisk do
+  before(:all) do
+    @test_env = TestEnvHelper.new(__FILE__)
+    @test_env.vcr_filter
+
+    @client_id     = @test_env[:azure_client_id]
+    @client_key    = @test_env[:azure_client_key]
+    @tenant_id     = @test_env[:azure_tenant_id]
+    @blob_uri      = @test_env[:blob_uri]
+
+    @test_env.ensure_recording_dir_exists
```

Comment:
Nope. This is the directory that contains the cassettes, so it shouldn't be removed.